ui/src/components/workflow/WorkflowMetaList.js

```diff
@@ -1,151 +1,92 @@
 import { Link, browserHistory } from 'react-router';
-import {Button, Col, Grid, Panel, Row} from 'react-bootstrap';
+import { Button } from 'react-bootstrap';
 import {BootstrapTable, TableHeaderColumn} from 'react-bootstrap-table';
 import { connect } from 'react-redux';
 import { getWorkflowDefs } from '../../actions/WorkflowActions';
-import Typeahead from "react-bootstrap-typeahead";
 
 const WorkflowMetaList = React.createClass({
 
-    getInitialState() {
-        return {
-            name: '',
-            version: '',
-            workflows: [],
-            workflowsFiltered: [],
-            labels: [],
-            allLabels: []
-        }
-    },
-
-    componentWillMount(){
-        this.props.dispatch(getWorkflowDefs());
-    },
-
-    componentWillReceiveProps(nextProps){
-        this.state.workflows = nextProps.workflows;
-        this.state.workflowsFiltered = nextProps.workflows;
-        this.state.allLabels = this.setAllLabels(nextProps.workflows);
-    },
-
-    setAllLabels(wfs) {
-        var tags = [];
-
-        for (let key in wfs) {
-            let wfsLabels = this.parseDescriptionRegex(wfs[key]);
-            if (!wfsLabels.length) {
-                continue;
-            }
-            tags = tags.concat(wfsLabels);
-        }
-        tags = _.uniq(tags);
-        return tags;
-    },
-
-    handleSearch() {
-        var wfs = this.state.workflows;
-        var labels = this.state.labels;
-
-        let filter = [];
-        for (let key in wfs) {
-            let wfsLabels = this.parseDescriptionRegex(wfs[key]);
-            if (!wfsLabels.length) {
-                continue;
-            }
-            if (labels.every(r=> wfsLabels.includes(r))){
-                filter.push(wfs[key]);
-            }
-        }
-        if (!labels.length){
-            filter = wfs;
-        }
-
-        this.setState({
-            workflowsFiltered: filter
-        })
-    },
-
-    //Parse Description to get name of LABELS
-    parseDescriptionRegex(workflow){
-        var allLabelsPattern = /- [A-Z0-9, ]*/gi;
-        var labelsPattern = /[A-Z0-9]+/gi;
-        let str = workflow.description;
-        let labels = str.match(allLabelsPattern);
-        if (!Array.isArray(labels) || !labels.length)
-            return [];
-        return labels[0].match(labelsPattern);
-    },
-
-    handleSubmit(labels) {
-        this.state.labels = labels;
-        this.handleSearch();
-    },
-
-    showAllWorkflows(){
-        this.setState({
-            workflowsFiltered: this.state.workflows,
-            labels: []
-        })
-    },
-
-    render() {
-        var wfs = this.state.workflowsFiltered;
-
-        function jsonMaker(cell, row) {
-            return JSON.stringify(cell);
-        }
-
-        function taskMaker(cell, row) {
-            if(cell == null){
-                return '';
-            }
-            return JSON.stringify(cell.map(task => {return task.name;}));
-        }
```

Comment:
This part of code shouldn't be overwritten. You probably weren't checked out on masters current version.